pyccel/ast/cwrapper.py

```diff
@@ -1,3 +1,7 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
 # pylint: disable=missing-function-docstring, missing-module-docstring/
```

Comment:
This must have slipped through a review. It shouldn't be here!

@MenoIy `git blame` says you added this line. Could you provide a module docstring please?